Makefile

```diff
@@ -221,6 +231,12 @@ bundle: update-mgr-env manifests kustomize operator-sdk rename-csv build-prometh
 	cd config/default && $(KUSTOMIZE) edit set namespace $(OPERATOR_NAMESPACE)
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	cd config/default && $(KUSTOMIZE) edit set image rbac-proxy=$(RBAC_PROXY_IMG)
+	cd config/manifests/bases && \
+		rm -rf kustomization.yaml && \
+		$(KUSTOMIZE) create --resources $(BUNDLE_PACKAGE).clusterserviceversion.yaml && \
+		$(KUSTOMIZE) edit add annotation --force 'olm.skipRange':"$(SKIP_RANGE)" && \
+		$(KUSTOMIZE) edit add patch --name $(BUNDLE_PACKAGE).v0.0.0 --kind ClusterServiceVersion\
```

Comment:
- i think it's better to have `$(BUNDLE_PACKAGE).v$(VERSION)` rather than `$(BUNDLE_PACKAGE).v0.0.0`

Reply:
This version is coming from this file [1] which has to be exactly the same always.

[1]. https://github.com/red-hat-storage/lvm-operator/blob/main/config/manifests/bases/clusterserviceversion.yaml.in#L7